### ISSUE TITLE
add "See All" for transactions

### ISSUE THREAD

**Issue by nicolasburtey** [GaloyMoney/galoy-mobile#2778]
https://www.figma.com/proto/rH7A7NquSsXEH7v8eGuYM7/Blink-(BBW)-Mobile-App-Design?page-id=5871%3A96345&type=design&node-id=7693-1871&viewport=-980%2C1006%2C0.74&t=47NqV5btpaxwNBBC-1&scaling=min-zoom&starting-point-node-id=6160%3A97622&show-proto-sidebar=1&mode=design

**Comment by designsats** [comment]
Simple solution to problem is to change "Transactions" (non intuitive) to "All transactions" and color it accent color: 

![home - light mode](https://github.com/GaloyMoney/galoy-mobile/assets/17384787/f67f6527-f1dc-400b-9999-0f3d9dce512e)

**Comment by MaxwellDG** [comment]
Hi I'd like to work on this. Should I wait for this to be assigned/approved? Not sure what the system you use is for green-lighting proposed changes.

PS I've tried to contact someone on Mattermost over a month ago to no avail. Is there someone I could talk to about this project? I'm a Canadian but I used to live in El Salvador for a summer. I used this app all the time in El Zonte. I'd be really cool to chat with someone about how I could get further involved. @nicolasburtey @designsats
